### PR TITLE
BETA: Register mosahasnmosa.is-a.dev

### PR DESCRIPTION
**Pull request by mosageneral**
Added `mosahasnmosa.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/hamzamosa.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mosageneral",
+        "email": "mosageneral@gmail.com"
+    },
+    "record": {
+        "URL": "takedin.net"
```
Comment (Stef-00012) [inline domains/hamzamosa.json:7]: Invalid URL record (missing protocol)

File: domains/mosahasn.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mosageneral",
+        "email": "mosageneral@gmail.com"
+    },
+    "record": {
+        "A": ["150.230.126.191"]
```
Comment (Stef-00012) [inline domains/mosahasn.json:7]: I cannot reach your website
"150.230.126.191 took too long to respond."

File: domains/mosahasnmosa.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mosageneral",
+        "email": "mosageneral@gmail.com"
+    },
+    "record": {
+        "A": ["150.230.126.191"]
```
Comment (Stef-00012) [inline domains/mosahasnmosa.json:7]: I cannot reach your website
"150.230.126.191 took too long to respond."

File: domains/mosaodoo.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mosageneral",
+        "email": "mosageneral@gmail.com"
+    },
+    "record": {
+        "A": ["150.230.126.191"]
```
Comment (Stef-00012) [inline domains/mosaodoo.json:7]: I cannot reach your website
"150.230.126.191 took too long to respond."

File: domains/odoomosa.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "mosageneral",
+        "email": "mosageneral@gmail.com"
+    },
+    "record": {
+        "A": ["150.230.126.191"]
```
Comment (Stef-00012) [inline domains/odoomosa.json:7]: I cannot reach your website
"150.230.126.191 took too long to respond."

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**